Mark deprecated SSL settings as obsolete (#147)
logstash-plugins/logstash-output-http: docs/index.asciidoc

This plugin supports the following configuration options plus the <<plugins-{type}s-{plugin}-common-options>> described later.

> NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. Please see the
> <<plugins-{type}s-{plugin}-obsolete-options>> for more details.

Reviewer: ```suggestion
NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. 
Please check out the
<<plugins-{type}s-{plugin}-obsolete-options>> for details.
```

> NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. Please see the
> <<plugins-{type}s-{plugin}-obsolete-options>> for more details.

Reviewer: ```suggestion
NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. 
Please check out <<plugins-{type}s-{plugin}-obsolete-options>> for details.
```

Reviewer: Asciidoc best practice is to start each sentence on a new line. The modular approach makes updating docs source easier to review and change.

See https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity(int)[these docs for more info]

[id="plugins-{type}s-{plugin}-obsolete-options"]
==== HTTP Output Obsolete Configuration Options

> WARNING: As of version `6.0.0` of this plugin, the following configuration options are no longer available,
> and have been replaced by the following options:

Reviewer: ```suggestion
WARNING: As of version `6.0.0` of this plugin, some configuration options have been replaced.
The plugin will fail to start if it contains any of these obsolete options. 
```
